db/migrate/20200524074928_create_service_resource_attachments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateServiceResourceAttachments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :service_resource_attachments do |t|
+      t.references :storage_service, :type => :bigint, :index => true
+      t.references :storage_resource, :type => :bigint, :index => true
+      t.boolean :compliant
```

Comment:
What is `compliant`?  I'm concerned this will be confused with our existing compliances modeling.  Perhaps we should use `ems_compliant` or `native_compliant` instead to differentiate?  @agrare Thoughts?

Reply:
Who is evaluating this compliance?  Is it the native system or MIQ?

If it is the native system then yeah ems_compliant would be less confusing, if it is an MIQ plugin then we should be using the existing compliance infrastructure not introduce new columns

Reply:
The compliance is evaluated by the Autosde management system. When False, this is to indicate that a specific resource that is already attached to a service no longer matches the capabilities defined by the service. So yes, ems_compliant is probably better

Reply:
we will remove the compliance field for this release